helm/charts/cloud-carbon-footprint/README.md

```diff
@@ -0,0 +1,119 @@
+| `client.ingress.hosts`                      | Hostnames for the client ingress                                                       | `[{ host: cloud-carbon-footprint.example.com }]`                                         |
+| `client.ingress.tls`                        | TLS configuration for the client ingress                                               | `[]`                                                                                     |
+| `client.pdb.enabled`                        | Enable Pod Disruption Budget for the client                                            | `true`                                                                                   |
+| `client.pdb.minAvailable`                   | Minimum number of available client pods                                                | `1`                                                                                      |
+
+#### Security Context for Client
+
+| Parameter                           | Description                                                | Default Value                          |
+|-------------------------------------|------------------------------------------------------------|----------------------------------------|
+| `client.deployment.securityContext.capabilities.add`    | Capabilities to add                                        | `[CHOWN, SETGID, SETUID]`              |
+| `client.deployment.securityContext.capabilities.drop`   | Capabilities to drop                                       | `[ALL]`                                |
+
+### API Configuration
+
+| Parameter                                | Description                                                                             | Default Value                                                                            |
+|------------------------------------------|-----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------|
+| `api.configmap`                          | Configuration map for the API                                                           | `{}`                                                                                     |
+| `api.deployment.replicaCount`            | Number of replicas for the API deployment                                               | `2`                                                                                      |
+| `api.deployment.image.repository`        | Docker image repository for the API                                                     | `docker.io/cloudcarbonfootprint/api`                                                     |
+| `api.deployment.image.tag`               | Docker image tag for the API                                                            | `release-2022-10-17@sha256:fbd781ea1b042f7fe29592d96e7d987c89400856efc7a90e1d9900584797e9ec` |
```

Comment:
I believe you updated default value was to release-2024-02-11

Reply:
Yes updated in the readme file